config/quickLookIsr.py

```diff
@@ -1,6 +1,7 @@
-config.doWrite = False  # this task writes separately, no need for this
 config.doSaturation = True  # saturation very important for roundness measurement in qfm
-config.doSaturationInterpolation = True
-config.overscan.fitType = "MEDIAN_PER_ROW"
-config.overscan.doParallelOverscan = True
 config.brighterFatterMaxIter = 2  # Uncomment this to remove test warning
+config.doDeferredCharge = False  # no calib for this yet
+config.doBootstrap = False
+config.doApplyGains = False
```

Comment:
Unless you expect to never have a PTC, I would strongly suggest this be set to `doApplyGains=True`.  The new ISR defaults largely assume that everything has had gains applied so units are in electrons.  We also should have CTI calibrations in the next set, but that correction is far less likely to have visual impact on the images than `doApplyGains` (which should help remove amp offsets).